app/controllers/static_controller.rb

```diff
@@ -1,5 +1,13 @@
 class StaticController < ApplicationController
   def index
+    use_new_styles
+
+    @press = {
+      'fast-company' => 'http://fastcompany.com',
+      'forbes' => 'http://forbes.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.